cmd/kube-apiserver/app/server.go

```diff
@@ -430,6 +430,7 @@ func BuildGenericConfig(s *options.ServerRunOptions) (*genericapiserver.Config,
 			versionedInformers.Core().V1().Services().Lister(),
 		)
 	}
+	genericConfig.SharedInformerFactory = versionedInformers
```

Comment:
what happened here before without that line?

Reply:
I would like to know will this be fixed?

Reply:
@karlvlam Sorry for the delaying.

ping @caesarxuchao @deads2k PTAL. Any comments on the above suggestions from [me](https://github.com/kubernetes/kubernetes/pull/50674#discussion_r133197704) and [@sttts](https://github.com/kubernetes/kubernetes/pull/50674#discussion_r133200809).